src/krb5/_creds.pyi

```diff
@@ -16,6 +17,24 @@ class TicketTimes(typing.NamedTuple):
     endtime: int
     renew_till: int
 
+class TicketFlags(enum.IntFlag):
```

Comment:
Would be great to add a docstring about this enum that links to the RFC where they are defined.